editor/src/clj/editor/gl/vertex2.clj

```diff
@@ -509,19 +325,19 @@
 
 (defrecord VertexBufferShaderLink [request-id ^VertexBuffer vertex-buffer shader]
   GlBind
-  (bind [_this gl render-args]
+  (bind [_this gl _render-args]
```

Comment:
Prefixing with and underscore `_` gets rid of the `unused parameter` warnings in IntelliJ.